app/routes/application.js

```diff
@@ -106,6 +106,10 @@ let ApplicationRoute = Route.extend(ApplicationRouteMixin, ModalHelper, SetupUse
   afterModel() {
     set(this.controllerFor('navigation'), 'allowSearch', false);
     $('#apploading').remove();
+    this.get('config.configDB').get('current_user').then((user) => {
+      let language = user.i18n || 'en';
```

Comment:
You mentioned `_setUserLanguageChoice(language)` saves a new record in the configDB.  I had a different understanding of how it works.  I believe `_setUserLanguageChoice(language)` first fetches the current_user from the configDB, i.e. `configDB.get('current_user')`.  That returns a promise, and when that's resolved successfully it passes a `user` object to the success callback.  That callback takes the user object and uses its `_id` attribute to send a `put` call to the `configDB`.  That `put` call looks like this:

    configDB.put({
            i18n: language,
            _id: user._id,
            _rev: user._rev,
            value: user.value
          });

Because I specified the `_id` attribute of the user, my understanding is that PouchDB will overwrite the existing record's attributes (i.e. an update operation) instead of creating a new record.  My intent was to follow the guide here:

https://pouchdb.com/guides/documents.html#updating-documents–correctly

As a tangent to this conversation, to make my intent clearer I refactored the code a bit in `_setUserLanguageChoice(langauge)` (now called `_setUserLanguage(language)` for succinctness).

I'm still learning how both HospitalRun and PouchDB work, so I could be wrong.  Happy to hop on a Slack chat or something if it's easier to talk that way.